h2o-core/src/main/java/water/rapids/RadixCount.java

```diff
@@ -1,5 +1,11 @@
 package water.rapids;
 
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Have you seen the BinaryMerge.java? It has like a gazillion of TODOs :)
Anyways, I actually wanted to implement all these renames sometime sooner than later, and all these comments are merely to serve as a discussion point on whether people think the rename is appropriate or not. Of course, I can make a JIRA on top too.